fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/GroupByTest.java

```diff
@@ -500,7 +500,7 @@ protected RecordMetaDataHook setupHookAndAddData(final boolean addIndex, final b
 
     @Nonnull
     private static FunctionKeyExpression bitBucketExpression(@Nonnull KeyExpression fieldExpression, int bucketSize) {
-        return Key.Expressions.function("bit_bucket", concat(fieldExpression, Key.Expressions.value(bucketSize)));
+        return Key.Expressions.function("bitmap_bucket_offset", concat(fieldExpression, Key.Expressions.value(bucketSize)));
```

Comment:
This is currently failing because of an unintended behaviour in `TreeLike#replaceLeavesMaybe` which is addressed in this PR #2885.

Reply:
has pulled the fix and the problem was fixed.